docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk"
+    },
+```
+
+Also under the `client` object add another property called `clientSignedCert` and under this add a property called `path` that has a string variable leading to the identities signed certificate. This can be found in the network, in this example it is found under **fabric-samples** -> **test-network** -> **organizations** -> **peerOrganizations** -> **org1.example.com** -> **users** -> **Admin@org1.example.com** -> **msp** -> **signedcerts** -> **admin@org1.example.com-cert.pem**
+
+This is what should be added to the `client`:
+
+``` bash
+"clientSignedCert": {
+    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem"
+    },
+```
+
+That is all you need to add to the `clients` object. It should now look like this: 
+
+```bash
+    "clients": {
+        "Admin@org1.example.com": {
+            "client": {
+                "credentialStore": {
+                    "path": "tmp/hfc-kvs/org1",
+                    "cryptoStore": {
+                        "path": "tmp/hfc-kvs/org1"
+                    }
+                },
+                "organization": "Org1",
+                "clientPrivateKey": {
+                    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk"
+                },
+                "clientSignedCert": {
+                    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem"
+                },
+                "connection": {
+                    "timeout": {
+                        "peer": {
+                            "endorser": "300"
+                        }
+                    }
+                }
+
+            }
+        }
+    },
+```
+
+### Channels Object
+
+The Fabric connector for Caliper requires assistance when creating connections to a Fabric network. A `channels` object must be provided that lists the smart contracts that may be interacted with. 
+
+Add an object called `channels` to the schema and a property named after the channel name. The default channel name, and the one used in this tutorial, is `mychannel`. Under the channel name, add two properties, one called `created` and the other `contracts`. 
+
+Set `created` to the boolean variable `true`. It should look like this:
+
+``` bash
+"created" : true,
+```
+
+`contracts` holds an array with two properties, `id` and `version`. `id` specifies contract ID; in this case it is `fabcar`. The version is that specific contract version; in this case it is `0.0.1`. It should look like this: 
+
+``` bash
+"contracts": [
+    {
+        "id":"fabcar",
+        "version":"0.0.1"
```

Comment:
Is version required if we aren't creating ?

Reply:
I think it's part of the TX proposal, but not sure what happens if you omit it.

Reply:
Version isn't required for the tx proposal as you can't select different versions of an instantiated contract on a channel. It would be useful for operational activities of install and instantiate though.